packages/console/src/components/RolesTransfer/components/TargetRoleItem/index.tsx

```diff
@@ -8,19 +8,27 @@ import * as styles from './index.module.scss';
 
 type Props = {
   role: RoleResponse;
+  type: RoleType;
   onDelete: () => void;
 };
 
-function TargetRoleItem({ role, onDelete }: Props) {
+function TargetRoleItem({ role, type, onDelete }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
-  const { name, usersCount } = role;
+  const { name, usersCount, applicationsCount } = role;
```

Comment:
Can you get `type` from `role`? If yes, then you don't have to pass `type` through params

Reply:
Good call

Reply:
Updated.